solana/programs/token-bridge-relayer/Cargo.toml

```diff
@@ -36,10 +40,11 @@ anchor-spl = { version = "0.30.1", git = "https://github.com/coral-xyz/anchor/",
 solana-price-oracle = { path = "../../../lib/relayer-infra-contracts/src/solana/programs/price-oracle" }
 
 #wormhole-anchor-sdk = { git = "https://github.com/wormhole-foundation/wormhole-scaffolding.git", rev = "1acd10e3cbeea0df06976b606bd24da2e8e849e1", features = ["token-bridge"] }
-wormhole-anchor-sdk = { git = "https://github.com/real-felix/wormhole-scaffolding.git", branch = "bump-anchor", default-features=false,features = [
+wormhole-anchor-sdk = { git = "https://github.com/real-felix/wormhole-scaffolding.git", branch = "bump-anchor", default-features = false, features = [
```

Comment:
we'll need to find a way to replace the `solana-devnet` feature on this dependency when building for mainnet